src/app/core/order/order.service.spec.ts

```diff
@@ -16,24 +16,24 @@ describe('OrderService', () => {
   describe('compareAsc', () => {
 
     it('should sort in ascending order', () => {
-      expect(orderService.compareAsc(ITEM_1, ITEM_2)).toBe(-1);
```

Comment:
how come this change? According to the describe this is specifically testing the compareAsc method. Also, sortAsc is deprecated